k8s/upgrade/src/bin/upgrade-job/upgrade/path.rs

```diff
@@ -17,7 +17,7 @@ use std::{fs, path::PathBuf};
 use utils::API_REST_LABEL;
 
 /// Validates the upgrade path from 'from' Version to 'to' Version for the Core helm chart.
-pub(crate) fn is_valid_for_core_chart(from: &Version) -> Result<bool> {
+pub(crate) fn upgrade_path_is_valid_for_core_chart(from: &Version) -> Result<bool> {
```

Comment:
way too long :) I'd vote to keep as it was

Reply:
Reverted back.